src/udev/cdrom_id/cdrom_id.c

```diff
@@ -249,8 +249,7 @@ static int cd_inquiry(int fd) {
         }
 
         if ((inq[0] & 0x1F) != 5) {
-                log_debug("not an MMC unit");
-                return -1;
+                return log_debug_errno(SYNTHETIC_ERRNO(1), "not an MMC unit");
         }
```

Comment:
Here too, and below.

Reply:
Hmm, thinking about this specific case, I'd say `errno` would have a pretty irrelevant value here. Shouldn't we use `SYNTHETIC_ERRNO(EINVAL)` instead?

Reply:
You're right, `errno` makes not sense here.